build/ci_utils/src/programs/wasm_opt.rs

```diff
@@ -65,7 +65,7 @@ impl std::str::FromStr for Version {
 
 impl IsVersion for Version {
     fn find_in_text_internal(text: &str) -> Result<Self> {
-        let number_regex = regex::Regex::new(r#"\d+"#)?;
+        let number_regex = regex::Regex::new(r"\d+")?;
```

Comment:
Why this change? The previous code was nicer and it explicitly stated a raw string here.

Reply:
New clippy lint, we probably need to disable it, it's really weird.